vocab/security/vocabulary.yml

```diff
@@ -39,9 +39,18 @@ class:
       - label: vc-data-integrity
         url: https://www.w3.org/TR/vc-data-integrity/#dataintegrityproof
 
+  - id: Multikey
+    label: Multikey Verification Method
+    upper_value: sec:VerificationMethod
+    comment: Verification method to be used with, for example, data integrity proof cryptographic suites, such as the eddsa-2022 cryptographic suite. See the <a href="https://www.w3.org/TR/vc-di-eddsa/#multikey">EdDSA Cryptosuite v2022 specification</a> for further details.
+    see_also:
+      - label: eddsa-2022
+        url: https://www.w3.org/TR/vc-di-eddsa/#multikey
+
   - id: Ed25519Signature2020
     label: ED2559 Signature Suite, 2020 version
-    upper_value: sec:Proof
+    upper_value: sec:VerificationMethod
+    deprecated: true
```

Comment:
Why ?

Reply:
Why on which of the two statements?

For both cases I did not invent anything, just followed what is in the spec and/or in the other PR-s, that is all.

@msporny @dlongley ?